internal/state/indexer/indexer_service.go

```diff
@@ -83,6 +83,9 @@ func (is *Service) publish(msg pubsub.Message) error {
 	if curr.Pending == 0 {
 		// INDEX: We have all the transactions we expect for the current block.
 		for _, sink := range is.eventSinks {
+			if sink.Type() == NULL {
```

Comment:
We won't get to this point in the code if indexing is disabled (see line 127 in `OnStart`).  

It should be fine to `continue` here, but we should not `return`. I don't object to the check, but I doubt it will matter much: In practice there is only ever zero or one indexer in the slice. 

(I'm not sure why we added the option for multiple, possibly as a migration aid?)

Reply:
makes sense, wanted to sanity check this before opening for r4r, thanks for the comment. 

Not really sure either, I don't see anyone ever using two, feel like this could cause a bug somewhere else?